### ISSUE TITLE
Hi Shani

### ISSUE THREAD

**Issue by ramboo80** [allouch89/ShaniXBMCdatabase#100]
Hi Shani
Sorry to write here but i don't know what happened i could't write a replay in the forum..
Can u help me please i want to install Shahid arabic  (bought new android tv box) and i could not get it from fusion repo it seems to be gone there...The other method of installation link in the first page gives me these three files 
directory/
addons.xml	
addons.xml.md5on
i dont know what to do with them..
please help

**Comment by emadmahdi** [comment]
@ramboo80 ...

If i understands your problem correctly ... i think you are accessing the "Fusion Repository" using "Kodi File Manager" while the correct method is to access "Fusion Repository" using the "Kodi Addon Installer" ... or simply you can follow the following steps:

1- install the repository:
go to Kodi Home --> System --> Settings --> Add-ons --> Install from zip file --> Fusion --> kodi-repos --> International --> arabic-repository.arabic.xbmc-addons1.0.1.zip

2- install the addon:
go to Kodi Home --> System --> Settings --> Add-ons --> Install from repository --> Shahid Addon Repository --> Video Addon --> Shaid Arabic

**Comment by ramboo80** [comment]
Hi Emad 
Thank you very much for the help.
I did exactly as you said but i got only the version 3.3.1 wich is not working and has allot of problems.
But i could sort it out by another way..I transfered files from my phone (wich i have shahid arabic 3.8.3 on it) transfered the Android/obb/kodi/..... and it worked so i got all my channels and favorites backup.
But i want to ask you Emad what about Shani after the Zem issue? everything seems not working.The forum i can not write a replay in the shahid mbc addon thread..Will shani continue with Shahid arabic addon?
